nl_NL/LC_MESSAGES/youtube.po

```diff
@@ -4,191 +4,191 @@ msgstr ""
 
 msgid "YOUTUBE_STATUS_COMPLETE"
-msgstr "Your upload is complete!"
+msgstr "Your upload is Klaar!"
 
 msgid "YOUTUBE_STATUS_FAILED"
-msgstr "An issue occured. For more details, visit https://flipnot.es/ytsetup"
+msgstr "Een error trad op. voor meer informatie, ga naar https://flipnot.es/ytsetup"
 
 msgid "RECONNECT_NEEDED"
-msgstr "The connection to your YouTube channel is out of date."
+msgstr "Je koppeling met je youtube kanaal is veroudert"
 
 msgid "BTN_START_UPLOAD"
 msgstr "Start Upload"
 
 msgid "YOUTUBE_ESTIMATED_DURATION"
-msgstr "Estimated video duration:"
+msgstr "Geschatte videoduur:"
 
 # Extra features provided by Sudomemo Plus (see flipnoteOptions.po)
 msgid "PLUS_FEATURES"
-msgstr "Plus Features"
+msgstr "Plus Functies"
 
 msgid "PLUS_FEATURES_NOTE"
-msgstr "Change these options from your Flipnote details page."
+msgstr "Wijzig deze opties op uw Flipnote-detailspagina."
 
 msgid "FLIPNOTE_SMOOTHING"
-msgstr "Flipnote Smoothing"
+msgstr "Flipnote Vezachting"
 
 msgid "AUDIO_ENHANCEMENT"
-msgstr "Audio Enhancement"
+msgstr "Audio verbetering"
 
 msgid "ON"
-msgstr "On"
+msgstr "Aan"
 
 msgid "OFF"
-msgstr "Off"
+msgstr "Uit"
 
 msgid "YOUTUBE_TERMS_AGREEMENT_NOTE"
-msgstr "By submitting your videos to YouTube, you acknowledge that you agree to YouTube's Terms of Service and Community Guidelines."
+msgstr "Door je video's te uploaden op YouTube, ga je ermee akkoord dat je akkoord gaat met de servicevoorwaarden en communityrichtlijnen van YouTube."
 
 # We may need to limit uploads per day depending on our API quota usage rate
 msgid "YOUTUBE_UPLOAD_LIMIT"
-msgstr "You can currently upload %s videos to YouTube per day."
+msgstr "Je kunt alleen %s videos op youtube uploaden per dag"
 
 msgid "YOUTUBE_UPLOAD_LIMIT_PLUS_LIMIT_NOTE"
-msgstr "Sudomemo Plus members can upload %s videos per day."
+msgstr "Sudomemo Plus Gebruiken kunnen %s videos per dag uploaden."
 
 msgid "YOUTUBE_UPLOAD_LIMIT_PLUS_UNLIMITED_NOTE"
-msgstr "Sudomemo Plus members can upload an unlimited number of videos each day."
+msgstr "Sudomemo plus gebruiken kunnen oneindig veel videos uploaden per dag"
 
 msgid "YOUTUBE_UPLOAD_LIMIT_CURRENT"
-msgstr "You have uploaded %s videos so far today."
+msgstr "Je hebt al %s videos geupload vandaag"
 
 msgid "YOUTUBE_UPLOAD_LIMIT_EXCEEDED"
-msgstr "You can't upload any more videos until tomorrow."
+msgstr "Je kan niet meer videos uploaden vandaag"
 
 # If the user's already uploaded it...
 msgid "FLIPNOTE_ALREADY_UPLOADED"
-msgstr "You've already uploaded this Flipnote to YouTube."
+msgstr "Je hebt deze flipnote al naar youtube geupload"
 
 msgid "YOUTUBE_ID"
 msgstr "YouTube ID"
 
 msgid "BTN_UPLOAD_AGAIN"
-msgstr "Upload Again"
+msgstr "Upload Opnieuw"
 
 msgid "YOUTUBE_STATUS_PAUSED_GLOBAL_QUOTA"
-msgstr "Our global usage quota has been exceeded, so the upload may be significantly delayed."
+msgstr "Ons algemene upload quota is overschreden uploads kunnen langer duren. sorry!"
```

Comment:
This shouldn't be 2 sentences, and it isn't apologizing. It's stating a fact that we have no more quota so it may be a while until the video will upload.